.github/workflows/dest_azb.yml

```diff
@@ -0,0 +1,81 @@
+name: Destination Plugin Azure Blob Storage Workflow
```

Comment:
Is there a reason this file is called `dest_azb` rather than `dest_azblob` like the plugin?

Reply:
Yeah, fixed it.